### ISSUE TITLE
In the demo, only single-tower structures are shown. Are there examples of twin-tower structures, similar to Sentence-BERT?

### ISSUE THREAD

**Issue by IcanDoItL** [PAIR-code/lit#1535]

**Comment by RyanMullins** [comment]
This is probably a better question for @iftenney to answer, but I don't recall ever having a Sentence-BERT-like example in our demos.

Are you looking to analyze such a model with a specific UI module, or just generally explore how LIT could be used to evaluate such models?

**Comment by IcanDoItL** [comment]
I am looking to analyze such a model with a specific UI module. I hope the score is cosine similarity,not logits(is_regression) or softmax
